### ISSUE TITLE
"Default" option not working for Checkbox fields

### ISSUE THREAD

**Issue by AndrewMac** [verbb/formie#563]
**Description**
I've just tried to add a default option for checkbox fields, but it didn't work.
The option shows as selected in the Formie preview, but it is not selected on the site itself.
Tested with radio buttons, and things work there okay—so just checkboxes :-)

**Steps to reproduce**
1. Add a checkbox field in Formie, with a few options
2. Choose one of these fields to be "default"
3. Check Formie preview: the default option is selected
4. Save the form and check the site: the default option is NOT selected

### Form settings
Please pick from the options below.

- Multi-page form: No
- Submission Method: Ajax
- Client-side Validation: Yes
- Custom Form Templates: No

**Additional info**
- Plugin version: 1.4.14
- Craft version: Pro 3.7.11
- Multi-site: Yes

**Additional context**

**Comment by engram-design** [comment]
Fixed for the next release. To get the fix early, change your `verbb/formie` requirement in `composer.json` to:

```
"require": {
  "verbb/formie": "dev-craft-3 as 1.4.15",
  "...": "..."
}
```

Then run `composer update`.

**Comment by AndrewMac** [comment]
Thanks so much for the super-quick work in getting that fixed! :-)

**Comment by engram-design** [comment]
Fixed in [1.4.16](https://github.com/verbb/formie/releases/tag/1.4.16)
